
#Analog Sensors
int32 BatteryVoltagemV
int32 BatteryCurrentmA
int32 BatteryTemperaturemC
int32 ExternalVoltagemV
int32 AccelerometerXmG
int32 AccelerometerYmG
int32 AccelerometerZmG
int32 VacuumCurrentmA
int32 SideBrushCurrentmA
int32 MagSensorLeftVAL
int32 MagSensorRightVAL
int32 WallSensormm
int32 DropSensorLeftmm
int32 DropSensorRightmm

#Digital sensors
bool SNSR_DC_JACK_IS_IN
bool SNSR_DUSTBIN_IS_IN
bool SNSR_LEFT_WHEEL_EXTENDED
bool SNSR_RIGHT_WHEEL_EXTENDED
bool LSIDEBIT
bool LFRONTBIT
bool LLDSBIT
bool RSIDEBIT
bool RFRONTBIT
bool RLDSBIT
